### PR TITLE
Translation: chapter 6

### PR DESCRIPTION
**Pull request by stephien**

### CODE REVIEW

File: documentation/en/chapter_6.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+<br>
+<br>
+
+## 6.5 MCU (Interface)
+
+For the selection of the microcontroller, we have focused on the  _Espressif_ controllers, as this is a brand that offers a variety of devices with incredible features such as Bluetooth and Wi-Fi at very low cost, which another brand could hardly offer in addition to the extensive documentation that exist for this device. 
+
+Benefits of the ESP32 family:
+- Economical.
```
Comment (stephien) [inline documentation/en/chapter_6.md:75]:
```suggestion
- Low-cost.
```

File: documentation/en/chapter_6.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+## 6.5 MCU (Interface)
+
+For the selection of the microcontroller, we have focused on the  _Espressif_ controllers, as this is a brand that offers a variety of devices with incredible features such as Bluetooth and Wi-Fi at very low cost, which another brand could hardly offer in addition to the extensive documentation that exist for this device. 
+
+Benefits of the ESP32 family:
+- Low-cost.
+- Extensive documentation available.
+- Examples on the Arduino platform that can be easily carried into the espressif environment. 
+- Energy- efficient.
```
Comment (stephien) [inline documentation/en/chapter_6.md:78]:
```suggestion
- Energy efficient.
```

**Comment by luisan00** [comment]
Please @stephien rename `chapter_6.md` to `chapter_06.md`

File: documentation/en/chapter_6.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
+# 6. Locha Mesh Hardware 
+ 
+This section presents the parameters that were used to choose all the control and transmission devices currently used in Locha Mesh.
```
Comment (gustavosinbandera1) [inline documentation/en/chapter_6.md:3]:
```suggestion
This section aims to show parameters that were used to choose all control and transmission devices currently used in Locha Mesh. 
```

File: documentation/en/chapter_6.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
+# 6. Locha Mesh Hardware 
+ 
+This section presents the parameters that were used to choose all the control and transmission devices currently used in Locha Mesh. 
+
+
+## 6.1 Radio frequency module 
+
+<img src="imple_pic/RF-selection.svg"  height="650" width="450" align="left"/>
+
+To choose the radio module that contains the microcontroller (MCU), the RF and modulation needed to transmit on the 915 MHz frequency, some parameters were established:
```
Comment (gustavosinbandera1) [inline documentation/en/chapter_6.md:10]:
```suggestion
To choose the radio module that contains the microcontroller (MCU), the RF and modulation needed to transmit on  915 MHz frequency, some parameters were established:
```

**Pull request closed without merging**